similarity-data-api/src/main/java/org/phenotips/data/similarity/PatientSimilarityView.java

```diff
@@ -70,4 +70,20 @@
      *         match, with {@code 0} for patients with no similarities
      */
     double getScore();
+
+    /**
+     * Get the phenotypic similarity score for this patient match.
+     *
+     * @return the similarity score, between 0 (a poor match) and 1 (a good
```

Comment:
@buske I see that the Javadoc for `getScore()` (see above) uses `[-1, 1]` for the score range. Is that valid or deprecated?